exchanges/okex/okex_types.go

```diff
@@ -428,3 +428,11 @@ var WithdrawalFees = map[string]float64{
 	symbol.ZIL:   20,
 	symbol.ZIP:   1000,
 }
+
+// Balance defines returned balance data
+type Balance struct {
+	Available float64 `json:"available"`
+	Balance   float64 `json:"balance"`
+	Currency  string  `json:"currency"`
+	Hold      float64 `json:"hold "`
```

Comment:
This does not appear to match the API documentation [here](https://github.com/okcoin-okex/API-docs-OKEx.com/blob/master/API-For-Spot-EN/REST%20API%20for%20SPOT.md): 
Request
POST https://www.okex.com/api/v1/wallet_info.do
Response
`{
    "info": {
        "funds": {
            "free": {
                "btc": "0",
                "ltc": "0",
                "eth": "0"
            },
            "holds": {
                "btc": "0",
                "ltc": "0",
                "eth": "0"
            }
        }
    },
    "result": true
}`
 If I've misunderstood, sorry. Also hold has a space there

Reply:
You are right, it was supposed to return to another object then append to an array of Balance but returned an error, I actually haven't been able to test it. Good spotting I forgot about that. I Think we need to change to V3 API?